cmd/src/args/topic.rs

```diff
@@ -116,6 +122,12 @@ pub(crate) struct TopicUpdateArgs {
     /// ("none" or skipping parameter causes removal of expiry parameter in topic)
     #[arg(value_parser = clap::value_parser!(MessageExpiry))]
     pub(crate) message_expiry: Option<Vec<MessageExpiry>>,
+    /// New max topic size in bytes
+    /// ("none" or skipping parameter causes removal of max topic size parameter in topic)
+    /// Can't be lower than segment size in the config.
```

Comment:
Optional parameters must be defined as last one. IMO those 2 new parameters should be just options not positional arguments with default values 1 (or 0 ?) and None.